nix/pkg/default.nix

```diff
@@ -39,49 +39,70 @@ assert lib.assertMsg (stdenv.isLinux || !gamemodeSupport) "gamemodeSupport is on
       ]);
     };
 
-    nativeBuildInputs = [extra-cmake-modules cmake jdk17 ninja canonicalize-jars-hook];
+    postUnpack = ''
+      rm -rf source/libraries/libnbtplusplus
+      ln -s ${libnbtplusplus} source/libraries/libnbtplusplus
+    '';
+
+    nativeBuildInputs = [
+      cmake
+      ninja
+      extra-cmake-modules
+      jdk17
+      stripJavaArchivesHook
+    ];
+
     buildInputs =
       [
-        qtbase
-        qtnetworkauth
-        zlib
-        quazip
+        cmark
         ghc_filesystem
+        kdePackages.qtbase
+        kdePackages.qtnetworkauth
```

Comment:
```suggestion
```
i know this was added here before, but i think it should be fine in just the wrapper. i'll test tomorrow to see

Reply:
No, I believe qtnetworkauth is a build dependency. I just tried and it didn't build without it.

Reply:
We definitely do depend on that now since the authentation was refactored

Reply:
> I just tried and it didn't build without it

Ah, thought it might have been a runtime plugin. Thanks!